azerothcore/azerothcore-wotlk
# feat(Core/Player): Implement option to convert excess honor points int…

…o money



## Changes Proposed:
-  Implement an option to conver excess honor points into gold after changing the honor cap
- 

## SOURCE:


## Tests Performed:

- tested ingame
- 


## How to Test the Changes:


1. UPDATE characters SET money = 0, totalhonorpoints = 76000;
2. log ingame
3. check if your honor converted into gold
4. tip: set MaxHonorPointsMoneyPerPoint to 10 and you'll get 1 gold after logging in



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
